### PR TITLE
Feat : Create post.txt

### LINKED ISSUE #2: Feat : Create post.txt
## 목적
- 게시물 작성 기능 추가

## 세부 내용
- sns를 통해 게시물을 작성해 의견을 나눌 수 있습니다.

### PR DESCRIPTION
**Pull request by minseokiim**
close #2

### CODE REVIEW

**Comment by syb0127** [comment]
Good job!

**Review by syb0127 — COMMENTED** [review]
Good job~

File: post.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+게시물을 작성하는 기능입니다.
```
Comment (jongsujin) [inline post.txt:1]: 고생하셨습니다!

**Review by angrydeveloper — APPROVED** [review]
asd

**Review by syb0127 — APPROVED** [review]
I approve

**Pull request merged**